src/Core/Scheduler/Providers/ThreeDTiles_Provider.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Created On: 2016-11-8
+ * Class: ThreeDTiles_Provider
+ * Description:
+ */
+import * as THREE from 'three';
+import Provider from './Provider';
+import CacheRessource from './CacheRessource';
+import B3dmLoader from '../../../Renderer/ThreeExtended/B3dmLoader';
+import Fetcher from './Fetcher';
+import FeatureMaterial from '../../../Renderer/FeatureMaterial';
+
+
+export function ThreeDTilesIndex(tileset) {
```

Comment:
AFAIK variables in javascript can start with most unicode character. We *could* find a char that looks like a 3 in other language, it would be funny for everyone involved ;-)

Jokes apart, why not following jquery example and start with `$`? `$3DTiles` etc... It is somehow familiar for a js dev, and has the advantage of not looking like a private var.